vite.config.js

```diff
@@ -46,6 +49,11 @@ function serveLocalesPlugin() {
 }
 
 export default defineConfig({
+    define: {
+        '__APP_VERSION__': JSON.stringify(pkg.version),
+        '__APP_PRODUCTNAME__': JSON.stringify(pkg.productName),
+        '__APP_REVISION__': JSON.stringify(childProcess.execSync("git rev-parse --short HEAD").toString().trim()),
```

Comment:
This does not translate to path executable
```
C:\dev\betaflight-configurator>git rev-parse --short HEAD
d042471d
```

Reply:
The issue is sonarcloud wants the full path to git. It doesn't like that I've let the system search for it... it can just be reviewed as 'safe'.

Reply:
it's asking the path for git 🤕